bin/cli.js

```diff
@@ -183,30 +183,67 @@ const handleDeprecatedEnvVariables = options => {
   ) {
     uiDeprecatedTag(
       i18n(`${i18nKey}.handleDeprecatedEnvVariables.portalEnvVarDeprecated`, {
-        configPath: getConfigPath(),
+        configPath: getConfigFilePath(),
       })
     );
     process.env.HUBSPOT_ACCOUNT_ID = process.env.HUBSPOT_PORTAL_ID;
   }
 };
 
+function addConfigEnvironmentVariablesMiddleware(options) {
+  const { useEnv, config } = options;
+
+  if (useEnv) {
+    process.env[ENVIRONMENT_VARIABLES.USE_ENVIRONMENT_HUBSPOT_CONFIG] = true;
+  }
+
+  if (config) {
+    process.env[ENVIRONMENT_VARIABLES.HUBSPOT_CONFIG_PATH] = config;
+  }
+}
+
 /**
  * Auto-injects the derivedAccountId flag into all commands
  */
 const injectAccountIdMiddleware = async options => {
-  const { account } = options;
+  const { account: accountOption } = options;
 
   // Preserves the original --account flag for certain commands.
-  options.providedAccountId = account;
+  options.providedAccountId = accountOption;
```

Comment:
Is there any scenario where we'd want to support both the `--use-env` flag and also the `--account` option at the same time? I'm wondering if we should abort if that ever happens. Using `--use-env` is essentially the same thing as passing in the `--account` flag